modules/FvwmPager/FvwmPager.c

```diff
@@ -1844,30 +1840,14 @@ void ParseOptions(void)
     }
     else if (StrEquals(token, "Monitor")) {
 	    char	*mname;
-	    int		 output, mdw, mdh, vx, vy, vxmax, vymax, iscur;
-	    int		 x, y, w, h;
 	    int		 updated = 0;
 
 	    next = GetNextToken(next, &mname);
-	    sscanf(next, "%d %d %d %d %d %d %d %d %d %d %d %d",
-			    &output, &iscur, &mdw, &mdh, &vx, &vy, &vxmax, &vymax,
-			    &x, &y, &w, &h);
 
 	    TAILQ_FOREACH(m2, &fp_monitor_q, entry) {
 		    updated = 0;
 		    if (strcmp(m2->name, mname) == 0) {
-			    m2->x = x;
-			    m2->y = y;
-			    m2->w = w;
-			    m2->h = h;
-			    m2->output = output;
-			    m2->is_current = iscur;
-			    m2->virtual_scr.MyDisplayWidth = mdw;
-			    m2->virtual_scr.MyDisplayHeight = mdh;
-			    m2->virtual_scr.Vx = vx;
-			    m2->virtual_scr.Vy = vy;
-			    m2->virtual_scr.VxMax = vxmax;
-			    m2->virtual_scr.VyMax = vymax;
+                extract_monitor_config(m2,next); 
```

Comment:
Again, indentation seems off.  Also:

```suggestion
                extract_monitor_config(m2, next); 
```